ci/github/unit-testing/dynamic-sidecar.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+IFS=$'\n\t'
+
+install() {
+    bash ci/helpers/ensure_python_pip.bash;
+    pushd services/dynamic-sidecar; pip3 install -r requirements/ci.txt -r requirements/_tools.txt; popd;
+    pip list -v
+}
+
+codestyle(){
+    pushd services/dynamic-sidecar
+    echo "isort"
+    isort --check setup.py src/simcore_service_dynamic_sidecar tests
+    echo "black"
+    black --check src/simcore_service_dynamic_sidecar tests/
+    echo "pylint"
+    pylint --rcfile=../../.pylintrc src/simcore_service_dynamic_sidecar tests/
+    echo "mypy"
+    mypy src/simcore_service_dynamic_sidecar tests/ --ignore-missing-imports
+    popd
+}
```

Comment:
that is nice. can you follow-up on this, create an issue for having this on the other projects  as well?

Reply:
agreed.  I would also like to have it as a recipe in ``scripts/common-service.Makefile`` so it can be used everywhere. Basically, it is as simple as to adapt and move the recipes in ``services/dynamic-sidecar/Makefile``. 
I also would like the github-actions shared.

Reply:
OK, managed to unify the codestyle for the CI and development lifecycles